rhombus/private/entry-point-adjustment.rkt

```diff
@@ -41,8 +43,8 @@
          entry-point-adjustment-method-table
          #hasheq())
   #:guard (lambda (args wrap is-method? info)
-            (unless (and (list? args) (andmap identifier? args))
-              (raise-argument-error* 'entry_point_meta.Adjustment rhombus-realm "List.of(Identifier)" args))
+            (unless (and (listable? args) (andmap identifier? (to-list #f args)))
```

Comment:
`(to-list #f args)` is computed twice.  Actually, will it be better to use `to-treelist` here (and have a version of `andmap` that works with that)?

Reply:
I agree about using `to-treelist` and I made that change. I'm not sure about `Listable.of` versus `Listable.to_list && ....` here, so that's pending.

Reply:
HF, with `Listable.to_list` added as a converter annotation.

Reply:
HF^2